.github/workflows/bench.yml

```diff
@@ -23,7 +23,7 @@ env:
 jobs:
   bench:
     name: Benchmark (Throughput)
-    runs-on: self-hosted
+    runs-on: [self-hosted, linux, x64, high-performance]
```

Comment:
I think I need a bit more understanding on how these would be defined to review the PR.

Reply:
Basically, I think we should follow the convention in the GitHub docs here https://github.com/philips-labs/terraform-aws-github-runner, so it should be `[self-hosted, operating-system, hardware-architecture, other-custom-labels]`.

Reply:
Sounds good. Plus we will be labeling our existing runners as `high-performance` today.